### ISSUE TITLE
learning rate decay

### ISSUE THREAD

**Issue by aaronma2020** [huggingface/accelerate#94]
I want to know if I use learning rate decay , like this:
   
```
optimizer= accelerator.prepare(optimizer)
current_lr = learning_rate * decay_factor
for group in optimizer.param_groups:
        group['lr'] = current_lr
```
Should I do this for the main process or  all processes? 
Thanks

**Comment by sgugger** [comment]
You should do it in all processes, so that they all use the same learning rate, otherwise the copies of your model in each process will diverge.
